requirements.in

```diff
@@ -7,7 +7,7 @@ attrs<21
 # https://github.com/flyteorg/flyte/issues/1732.
 jsonschema<4
 pyyaml<6
-# A number of dependencies, including pandas and numpy, started dropping support for python 3.7 in their latest
-# releases (pandas>=1.4.0 and numpy>=1.22.0). More details in https://github.com/flyteorg/flyte/issues/2115.
+# A number of dependencies, including pandas and NumPy, started dropping support for Python 3.7 in their latest
+# releases (pandas>=1.4.0 and NumPy>=1.22.0). More details in https://github.com/flyteorg/flyte/issues/2115.
 pandas<1.4.0
-numpy<1.22.0
+NumPy<1.22.0
```

Comment:
Can we revert the changes in this file except for `Python`? `pandas and NumPy` somewhat looks odd, and `numpy` is the library to be installed; we need not modify its case.

Reply:
Sure